parlai/tasks/anli/build.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import parlai.core.build_data as build_data
+from parlai.core.build_data import DownloadableFile
+from .anli_constants import ANLI, ANLI_VERSION
+
+RESOURCES = [
+    DownloadableFile(
+        'https://dl.fbaipublicfiles.com/anli/anli_v0.1.zip',
+        'anli_v0.1.zip',
+        '16ac929a7e90ecf9093deaec89cc81fe86a379265a5320a150028efe50c5cde8',
+    )
+]
+
+
+def build(opt):
+    dpath = os.path.join(opt['datapath'], ANLI)
+    version = ANLI_VERSION
```

Comment:
nit: i'd prefer the version to be defined at the top of this file, rather than in the separate file.